package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "yarn lint:scripts && yarn lint:styles && yarn format:md",
     "lint:scripts": "eslint --fix \"**/*.js\"",
     "lint:styles": "stylelint --fix \"**/*.css\"",
-    "precommit": "lint-staged && yarn test:changes",
+    "precommit": "lint-staged && yarn test:changes && node scripts/generate-sitemap.js && git add static/sitemap.xml",
```

Comment:
Can you remove this from the precommit hook?

I'll add it as part of a release process. If it's in the precommit hook, the modification dates will change all the time. Every PR will end up having a diff, even if they didn't add a new page.

After that, I'll approve and merge!

Reply:
@kylemh  Done :D